src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,55 @@
+@use "../../mixins/button";
+@use "../../design-tokens/sizes.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ */
+
+$rhythm-space-generous: sizes.$rhythm_space_generous;
+$list-inline-gap: sizes.$list_inline_gap;
```

Comment:
Are these extra variables necessary here? I had them in my CodePen because I didn't have the actual design tokens to reference.

Also, the design tokens should be referenced with dashes rather than underscores.

Reply:
👋 @tylersticka 

> Are these extra variables necessary here?

I don't know that they are _necessary_, but I thought they were a convenience, since the same tokens are used on multiple lines in the same file and I thought it would offer developer convenience to be able to change values in a single location. 

Perhaps I should work to gain a better understanding of when it is appropriate to to create custom variables from tokens and when not to. 

For example, I observed this in `table.scss`:

https://github.com/cloudfour/cloudfour.com-patterns/blob/31e00c2cce7b3b6a42d9884dac0256a6a772dc18/src/components/table/table.scss#L6-L8

...and this in `calendar-date.scss`:

https://github.com/cloudfour/cloudfour.com-patterns/blob/31e00c2cce7b3b6a42d9884dac0256a6a772dc18/src/components/calendar-date/calendar-date.scss#L7-L9

Reply:
I think what's giving me pause is that the variables are the same name in both places. In the examples you provided, there's some abstraction happening... the `$table-` variables being a good example.

If these variables were named `$wp-button-gap: sizes.$list-inline-gap`, I think the intent would be a lot more clear.